src/fighter_state.rs

```diff
@@ -592,7 +607,7 @@ fn flopping(
                             BodyLayers::ENEMY_ATTACK
                         },
                         if is_player {
-                            BodyLayers::ENEMY
+                            BodyLayers::ENEMY + BodyLayers::BREAKABLE_ITEM
```

Comment:
I think this needs to be like this:

```suggestion
                            BodyLayers::ENEMY | BodyLayers::BREAKABLE_ITEM
```